.github/workflows/ci-base-tests.yml

```diff
@@ -1,6 +1,9 @@
 name: SMARTS CI Base Tests
 
-on: [push]
+on:
+  push:
+    branches:
+      - disabled
```

Comment:
This will disable `ci-base-tests` for the entire repository. Do not disable this workflow.

Reply:
Ohh okay okay